SimTKcommon/Simulation/include/SimTKcommon/internal/MeasureImplementation.h

```diff
@@ -1289,6 +1289,43 @@ template <class T>
 class Measure_Differentiate_Result {
 public:
     Measure_Differentiate_Result() : derivIsGood(false) {}
+
+    Xml::Element toXmlElement(const std::string& name) const {
+        static const int version = 1;
+        Xml::Element e("Measure_Differentiate_Result");
+        if (!name.empty()) e.setAttributeValue("name", name);
+        e.setAttributeValue("version", String(version));
+        e.appendNode(toXmlElementHelper(operand, "operand", true));
+        e.appendNode(toXmlElementHelper(operandDot, "operandDot", true));
+        e.appendNode(toXmlElementHelper(derivIsGood, "derivIsGood", true));
```

Comment:
Right -- we don't want any `:`s in XML tags. The element names here are arbitrary, basically just chosen so that the resulting XML is appealing. Nothing depends on them except the corresponding deserialization routine, written by the same person. The trick stuff is how we (de)serialize `Value<T>` -- the (de)serialization of the underlying T is kind of arbitrary.